### PR TITLE
Port 7441 GitLab Scheduled Ocean for Linear, Jenkins, APM and Terraform

### PR DESCRIPTION
**Pull request by phalbert**
# Description

Integrations:
- Linear
- Jenkins
- Dynatrace
- Sentry
- New Relic
- LaunchDarkly
- Terraform Cloud
- Jira

## Updated docs pages

Please also include the path for the updated docs

- docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md
- docs/build-your-software-catalog/sync-data-to-catalog/linear/linear.md
- docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/sentry.md
- docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/newrelic.md
- docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md
- docs/build-your-software-catalog/sync-data-to-catalog/feature-management/launchdarkly.md
- docs/build-your-software-catalog/sync-data-to-catalog/terraform-cloud/terraform-cloud.md

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md [hunk 1]
```diff
@@ -311,6 +311,70 @@ steps:
...
+| `OCEAN__PORT__CLIENT_ID`                         | Your port client id                                                                                                | ✅       |
+| `OCEAN__PORT__CLIENT_SECRET`                     | Your port client secret                                                                                            | ✅       |
+| `OCEAN__PORT__BASE_URL`                          | Your port base url, relevant only if not using the default port app                                                | ❌       |
+
+
+
+<br/>
+
+
+Here is an example for `dynatrace-integration.yml` pipeline file:
```
Comment (yairsimantov20) [inline docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md:340]: shouldnt it be gitlab-ci.yml? applies to all

**Review by yairsimantov20 — APPROVED** [review]
one comment 🌊

File: docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md (hunk 1 above)
Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md:340]: it should be

**Comment by phalbert** [comment]
Updated the Jira Integration too

**Review by MPTG94 — APPROVED** [review]
LGTM

**Pull request merged**